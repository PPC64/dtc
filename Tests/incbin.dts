/dts-v1/;

/ {
	data = [01 02], /incbin/("bytes"),  [ff fe];
};
