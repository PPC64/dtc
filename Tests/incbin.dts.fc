/dts-v1/;

/ {
	// CHECK: Cannot open binary include file
	data = [01 02], /incbin/("flibble_imaginary_file"),  [ff fe];
};
