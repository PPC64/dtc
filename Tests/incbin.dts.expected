/dts-v1/;

/  {

	data = <0x1020102 0x3040506 0x708fffe>;
};
